packages/react/src/core/packs.ts

```diff
@@ -65,6 +65,14 @@ export const print = {
 			printColorAdjust: 'exact',
 		},
 	},
+	// Hide URL rendered for links in print
+	hideLinkURL: {
+		'@media print': {
+			'&[href]:after': {
```

Comment:
pseudo elements are different than pseudo classes
```suggestion
			'&[href]::after': {
```

Reply:
Updated to match pseudo-element

https://github.com/agriculturegovau/agds-next/pull/1970/commits/8071497027261d563134d5c2faba3939192b261e